### DISCUSSION TITLE
Generating docs with Postman

### DISCUSSION THREAD

**Discussion by dizzydes** [vitalik/django-ninja#616] [Q&A]
Love the simplicity of this package.

Postman relies on Open API 3.0 conditions to auto generate docs (eg when a frontend dev wants to see endpoints). Does Ninja allow for this? Thanks

**Comment by vitalik** [comment]
@dizzydes do you have some example of the expected schema (and/or expected python type definition to describe it ) ?

**Comment by dizzydes** [comment]
Hey Vitalik

It mentions here that
*"If you are designing an API based on the OpenAPI 3.0 specification,
Postman automatically creates documentation based on your API definition."*
https://learning.postman.com/docs/publishing-your-api/documenting-your-api/#generating-api-documentation

Thanks,
Des

**Comment by vitalik** [comment]
@dizzydes 

Ah - yes - django-ninja generates openapi3 spec - so you can pass it to tools like Postman, Swagger, redocs, etc.
just pass  your-server/api/openapi.json to your doc generator

**Comment by dizzydes** [comment]
Amazing, thank you

On Sun 27 Nov 2022 at 10:24, Vitaliy Kucheryaviy ***@***.***>
wrote:

> @dizzydes <https://github.com/dizzydes>
>
> Ah - yes - django-ninja generates openapi3 spec - so you can pass it to
> tools like Postman, Swagger, redocs, etc.
> just pass your-server/api/openapi.json to your doc generator
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vitalik/django-ninja/discussions/616#discussioncomment-4245909>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACQHC5YBZXW5Y2PUHQ7KQILWKMSGRANCNFSM6AAAAAASLQA6QA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
Thanks,
Des
